pom.xml

```diff
@@ -240,7 +262,7 @@
                 </executions>
                 <configuration>
                     <skipITs>${it.skip}</skipITs>
-                    <forkCount>0</forkCount>
+                    <forkCount>1</forkCount>
```

Comment:
this is necessary, or just as a good idea?

Reply:
it is recommended to use forkCount > 0, also it allows us to spec env vars in failsafe plugin